script/console

```diff
@@ -12,8 +12,8 @@ if (!process.env.GITHUB_TOKEN) {
   process.exit(1);
 }
 
-const repl = require('repl').start('> ');
-const GitHubApi = require('github');
+import repl from 'repl').start('> ';
```

Comment:
I wasn't intending to touch this file actually, as I wasn't sure how to test it. I guess it must have been some global search / replace.

Can I test it by just running `script/console` ?

Reply:
i'm not sure i've ever even used it since taking over the project, so i cant say with confidence how to test it. i'd be ok with deferring changes to this file and just giving this file a `.cjs` extension for now

Reply:
Since it uses the `#!` it's defaulting to the CommonJS `require` syntax anyway. Do you think we still need to change the file extension?

Reply:
Looks like we do.

```
./script/console
node:internal/errors:464
    ErrorCaptureStackTrace(err);
    ^

TypeError [ERR_UNKNOWN_FILE_EXTENSION]: Unknown file extension "" for /Users/matt/git/github.com/probot/settings/script/console
    at new NodeError (node:internal/errors:371:5)
    at Object.getFileProtocolModuleFormat [as file:] (node:internal/modules/esm/get_format:87:11)
    at defaultGetFormat (node:internal/modules/esm/get_format:102:38)
    at defaultLoad (node:internal/modules/esm/load:21:14)
    at ESMLoader.load (node:internal/modules/esm/loader:359:26)
    at ESMLoader.moduleProvider (node:internal/modules/esm/loader:280:58)
    at new ModuleJob (node:internal/modules/esm/module_job:66:26)
    at ESMLoader.#createModuleJob (node:internal/modules/esm/loader:297:17)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:261:34)
    at async Promise.all (index 0) {
  code: 'ERR_UNKNOWN_FILE_EXTENSION'
}
```

Reply:
Resolved in [a158f25](https://github.com/probot/settings/pull/569/commits/a158f25c7ec928497e6e86a54c0a8dc13113a41b)